sdks/rust/src/alpha.rs

```diff
@@ -12,198 +12,116 @@
-        let cli = alpha_grpc::SdkClient::new(ch);
-        Alpha {
-            client: Arc::new(cli),
+    pub(crate) fn new(ch: Channel) -> Self {
+        Self {
+            client: SdkClient::new(ch),
         }
     }
 
     /// This returns the last player capacity that was set through the SDK.
-    /// If the player capacity is set from outside the SDK, use sdk.get_gameserver() instead.
-    pub fn get_player_capacity(&self) -> Result<i64> {
-        let req = alpha::Empty::new();
-        let count = self.client.get_player_capacity(&req).map(|c| c.count)?;
-        Ok(count)
-    }
-
-    /// This changes the player capacity to a new value.
-    pub fn set_player_capacity(&self, capacity: i64) -> Result<()> {
-        let mut c = alpha::Count::new();
-        c.set_count(capacity);
-        let res = self.client.set_player_capacity(&c).map(|_| ())?;
-        Ok(res)
-    }
-
-    /// This function increases the SDK’s stored player count by one, and appends this playerID to GameServer.status.players.ids.
-    /// Returns true and adds the playerID to the list of playerIDs if the playerIDs was not already in the list of connected playerIDs.
-    pub fn player_connect<S>(&self, id: S) -> Result<bool>
-    where
-        S: Into<String>,
-    {
-        let mut p = alpha::PlayerID::new();
-        p.set_playerID(id.into());
-        let res = self.client.player_connect(&p).map(|b| b.bool)?;
-        Ok(res)
-    }
-
-    /// This function decreases the SDK’s stored player count by one, and removes the playerID from GameServer.status.players.ids.
-    /// Will return true and remove the supplied playerID from the list of connected playerIDs if the playerID value exists within the list.
-    pub fn player_disconnect<S>(&self, id: S) -> Result<bool>
-    where
-        S: Into<String>,
-    {
-        let mut p = alpha::PlayerID::new();
-        p.set_playerID(id.into());
-        let res = self.client.player_disconnect(&p).map(|b| b.bool)?;
-        Ok(res)
-    }
-
-    /// Returns the current player count.
-    pub fn get_player_count(&self) -> Result<i64> {
-        let req = alpha::Empty::new();
-        let count = self.client.get_player_count(&req).map(|c| c.count)?;
-        Ok(count)
-    }
-
-    /// This returns if the playerID is currently connected to the GameServer.
-    /// This is always accurate, even if the value hasn’t been updated to the GameServer status yet.
-    pub fn is_player_connected<S>(&self, id: S) -> Result<bool>
-    where
-        S: Into<String>,
-    {
-        let mut p = alpha::PlayerID::new();
-        p.set_playerID(id.into());
-        let res = self.client.is_player_connected(&p).map(|b| b.bool)?;
-        Ok(res)
-    }
-
-    /// This returns the list of the currently connected player ids.
-    /// This is always accurate, even if the value has not been updated to the Game Server status yet.
-    pub fn get_connected_players(&self) -> Result<Vec<String>> {
```

Comment:
Tonic is async only so I opted to just remove all of the non-async methods. If the caller needs to call these async methods from a sync context, they can use something like [tokio::task::block_in_place](https://docs.rs/tokio/1.6.0/tokio/task/fn.block_in_place.html).